airflow/template/templater.py

```diff
@@ -155,6 +167,8 @@ def render_template(
             return self._render(template, context)
         if isinstance(value, ResolveMixin):
             return value.resolve(context)
+        if isinstance(value, LiteralValue):
+            return value.value
```

Comment:
Instead of adding a special case, it may be better to make LiteralValue inherit ResolveMixin and implement `resolve` to just return `value`. This would avoid surprises if the resolution logic is used elsewhere (can’t think of one but you never know with a large code base).

Reply:
I've come across this mixin, but the `iter_references` method appeared to be intended for a different use case. However, since `LiteralValue`, by definition, should have no references, I can implement it sensibly nonetheless.